websites/management/commands/markdown_cleaning/link_parser.py

```diff
@@ -88,7 +88,7 @@ class LinkParser(WrappedParser):
     angle-bracket destination variant treated properly.
     """
 
-    def __init__(self):
+    def __init__(self, recursive=False):
```

Comment:
`recursive=True` only seems to be used in a test, but I'm guessing it might be useful in the future?

Reply:
Yah... This PR added `recursive` as a configuration option on `LinkParser`. Before it was essentially the default. But it was problematic for the rootrelative_url rule in this PR, and not really relevant to the other rules.

I do think it will be useful for some things, so I left it in, but turned it off by default.